src/ScaleFactor.cc

```diff
@@ -423,3 +423,17 @@ double ScaleFactor::get_ScaleFactorError(double pt, double eta, int pType) {
 
   return get_ScaleFactorError(effData, effMC, errData, errMC);
 }
+
+double ScaleFactor::get_direct_ScaleFactorError(double pt, double eta, int pType){
+
+  std::string label = FindEtaLabel(eta, "data");
+  int pt_point = FindPtPoint(eff_data, label, pt, eta, pType); // when available, SF stored in eff_data (lazy implementation that should be improved)
+  double SF_error;
+
+  if (pt_point == -99){SF_error = 0;} // if pt is underflow
```

Comment:
Whenever ``` if (pt_point == -99)```, should we instead throw an exception and stop (including for functions you did not touch)?